auth/overview_snippets.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authsnippets contains Google Cloud authentication snippets.
+package authsnippets
+
+import (
+	"context"
```

Comment:
Imports should be added by `goimports` running on save, which usually puts them in two groups: standard lib and everything else (separated by a newline).

https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#imports

Reply:
Done :) Added to my IDE, thanks!